src/components/Map/__tests__/Map.spec.tsx

```diff
@@ -23,84 +25,123 @@ describe('<Map/> that renders a Map with markers', () => {
             lng: 4.9338793,
         },
         radius: 30000,
-    };
+    } as CircularMarker;
     const pointMarker = {
         center: {
             lat: 52.5112671,
             lng: 7.2535521,
         },
-    };
+    } as CircularMarker;
 
-    it('should render with default props', () => {
-        const wrapper = mount(<Map />);
-        expect(toJson(wrapper)).toMatchSnapshot();
-    });
     it('should set center and zoom when rendered with default props', () => {
-        mount(<Map />);
+        render(<Map />);
+
         expect(setZoomMock).toHaveBeenCalledTimes(1);
         expect(setCenterMock).toHaveBeenCalledTimes(1);
     });
+
     it('should fit bounds when rendered with defaultArea as address', () => {
         geocodeMock.mockImplementationOnce((req, cb) => {
             cb(geocodeResponse.results, geocodeResponse.status);
         });
-        mount(<Map defaultArea={{ address: 'nl' }} />);
+        render(<Map defaultArea={{ address: 'nl' }} />);
+
         expect(geocodeMock).toHaveBeenCalledTimes(1);
         expect(fitBoundsMock).toHaveBeenCalledTimes(1);
     });
+
     it('should add default highlight', () => {
-        mount(<Map defaultHighlight={NL_PATHS} />);
+        render(<Map defaultHighlight={NL_PATHS as GeoJSON.GeoJsonObject} />);
+
         expect(addGeoJsonMock).toHaveBeenCalledTimes(1);
     });
+
     it('should remove default highlight when markers added', () => {
-        const wrapper = mount(<Map defaultHighlight={NL_PATHS} />);
+        const view = render(<Map defaultHighlight={NL_PATHS as GeoJSON.GeoJsonObject} />);
+
         expect(addGeoJsonMock).toHaveBeenCalledTimes(1);
 
-        wrapper.setProps({ markers: [pointMarker, regionMarker] });
+        view.rerender(
+            <Map
+                defaultHighlight={NL_PATHS as GeoJSON.GeoJsonObject}
+                markers={[pointMarker, regionMarker]}
+            />
+        );
         // will be called as many times as the length of the returned array from addGeoJsonMock
         expect(removeDataMock).toHaveBeenCalledTimes(1);
     });
+
     it('should render with markers', () => {
-        const wrapper = mount(<Map markers={[pointMarker, regionMarker]} />);
-        expect(wrapper.find('Marker')).toHaveLength(2);
-        expect(wrapper.find('Circle')).toHaveLength(1);
+        render(<Map markers={[pointMarker, regionMarker]} />);
```

Comment:
can we test something here?

Reply:
Actually no. I'll remove the test.

Reply:
I think the test did have a purpose, as it checked, at least to some extent, that the props were correctly parsed. Now we cannot check React props anymore (based on other comments in this PR). I wonder if we should at least leave a small TODO note to point out that we missing tests for these parts of the code. 🤔

Reply:
We already have the test for checking `markers` props with the title `should fit map to markers'. The current test tested how the markers are rendered. For this kind of tests we need to mock Map component.